.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [2.6, 2.7, 3.0]
+        ruby-version: [2.7, 3.0, 3.1]
```

Comment:
Float `3.0` is misparsed as integer `3`, so `Test (3)` job used ruby 3.1.2.

https://github.com/groovenauts/capistrano-systemd-multiservice/actions/runs/3476345012/jobs/5811488575

![image](https://user-images.githubusercontent.com/162862/202084445-7ee7abe1-19b3-4fb7-bc99-e292f12cd765.png)

```suggestion
        ruby-version: [2.7, '3.0', 3.1]
```

OR 

```suggestion
        ruby-version: ['2.7', '3.0', '3.1']
```


ref: https://github.com/ruby/setup-ruby#matrix-of-ruby-versions